testcases/OpTestSTB.py

```diff
@@ -0,0 +1,623 @@
+        l_tb_bv = l_hw_tb_value & BMC_CONST.TB_MASK
+        if BMC_CONST.TB_MASK == l_tb_bv:
+            print BMC_CONST.TRUSTED_MODE_ON
+            return BMC_CONST.TRUSTED_MODE_ON
+        else:
+            print BMC_CONST.TRUSTED_MODE_OFF
+            return BMC_CONST.TRUSTED_MODE_OFF
+
+    ##
+    # @brief This function collects Secure boot mode information from OPAL
+    # messagelog and if found determines whether it is 'on' or 'off'.
+    # Returns SECURE_MODE_ON for Secure boot mode 'on'
+    # Returns SECURE_MODE_OFF for Secure boot mode 'off'
+    # Raises OpTestError when no information found
+    #
+    def secure_boot_mode(self):
+        l_secure_mode = self.string_get(self.l_data,
+                                        BMC_CONST.SECURE_MODE)
+        if l_secure_mode:
+            l_secure_mode_status = self.status_finder(l_secure_mode,
+                                                      BMC_CONST.ON,
+                                                      BMC_CONST.OFF)
+            if l_secure_mode_status == BMC_CONST.ON:
+                print BMC_CONST.SECURE_MODE_ON
+                return BMC_CONST.SECURE_MODE_ON
+            elif l_secure_mode_status == BMC_CONST.OFF:
+                print BMC_CONST.SECURE_MODE_OFF
+                return BMC_CONST.SECURE_MODE_OFF
+        else:
+            l_msg = "Secure mode enable/disable status not seen"
+            raise OpTestError(l_msg)
+
+    ##
+    # @brief This function collects Trusted boot mode information from OPAL
+    # messagelog and if found determines whether it is 'on' or 'off'.
+    # Returns TRUSTED_MODE_ON for Trusted boot mode 'on'
+    # Returns TRUSTED_MODE_OFF for Trusted boot mode 'off'
+    # Raises OpTestError when no information found
+    #
+    def trusted_boot_mode(self):
+        l_trusted_mode = self.string_get(self.l_data,
+                                         BMC_CONST.TRUSTED_MODE)
+        if l_trusted_mode:
+            l_trusted_mode_status = self.status_finder(l_trusted_mode,
+                                                       BMC_CONST.ON,
+                                                       BMC_CONST.OFF)
+            if l_trusted_mode_status == BMC_CONST.ON:
+                print BMC_CONST.TRUSTED_MODE_ON
+                return BMC_CONST.TRUSTED_MODE_ON
+            elif l_trusted_mode_status == BMC_CONST.OFF:
+                print BMC_CONST.TRUSTED_MODE_OFF
+                return BMC_CONST.TRUSTED_MODE_OFF
+        else:
+            l_msg = "Trusted mode enable/disable status not seen"
+            raise OpTestError(l_msg)
+
+    ##
+    # @brief This function collects CAPP measured information from OPAL
+    # messagelog and if found determines whether it is 'measured' or
+    # 'not measured'.
+    # Returns CAPP_PAR_MEASURED for CAPP partition measured
+    # Returns CAPP_PAR_NOT_MEASURED for CAPP partition not measured
+    # Raises OpTestError when no information found
+    #
+    def capp_measured(self):
```

Comment:
Change function name to opal_get_capp_measure_info. and just return True/ False for measure/not-measured cases.